itexico/Giphyur
# Front page trending with search

# What does this PR do?

Adds a front page with the trending gifs and a search bar to look for gifs

# How can one test it?

1. Switch to the branch

2. run npm start app should be running on http://localhost:8080/
